sdk/netapp/azure-resourcemanager-netapp/src/test/java/com/azure/resourcemanager/netapp/NetAppFilesManagerTests.java

```diff
@@ -34,15 +35,16 @@ public void beforeTest() {
         final TokenCredential credential = new AzurePowerShellCredentialBuilder().build();
         final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
 
-        netAppFilesManager = NetAppFilesManager.configure()
-            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
-            .authenticate(credential, profile);
-
         resourceManager = ResourceManager.configure()
             .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
             .authenticate(credential, profile)
             .withDefaultSubscription();
 
+        netAppFilesManager = NetAppFilesManager.configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .withPolicy(new ProviderRegistrationPolicy(resourceManager))
```

Comment:
I think this should work.

But for this RP, it was already registered via the explicit code (before I switched to this).

Reply:
tested on databricks, it works https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4250264&view=results